packages/base/src/panelview/leftpanel.tsx

```diff
@@ -34,16 +33,16 @@ export class LeftPanelWidget extends SidePanel {
     this.addWidget(properties);
 
     this._handleFileChange = () => {
-      header.title.label = this._currentContext?.localPath || '-';
+      header.title.label = this._currentContext?.filePath || '-';
     };
 
     options.tracker.currentChanged.connect((_, changed) => {
       if (changed) {
         if (this._currentContext) {
           this._currentContext.pathChanged.disconnect(this._handleFileChange);
         }
```

Comment:
Should we remove these? Since the `model.filePath` is not something that changes?

```suggestion
```

Reply:
it does change when you rename the jcad file

Reply:
Indeed. Since the model has a `filePath` property, how about we add a `pathChanged` getter on the model which returns the context `pathChanged` signal?